devai-cli/README.md

```diff
@@ -37,11 +37,15 @@ devai review security -c ../sample-app/src/main/java
 
 devai review code -c ../sample-app/src/main/java/anthos/samples/bankofanthos/balancereader/BalanceReaderController.java
 
-devai review testcoverage -c ../sample-app/src/main/java/anthos/samples/bankofanthos/balancereader
+devai review testcoverage -c ../sample-app/src
```

Comment:
switched to top level `src` folder so that we provide both code and tests for analysis.